tests/Cooked/Attack/DupTokenSpec.hs

```diff
@@ -144,8 +143,8 @@ tests =
                       { txSkelLabel = Set.singleton $ TxLabel DupTokenLbl,
                         txSkelMints = txSkelMintsFromList [(pol, emptyTxSkelRedeemer, tName1, 2)],
                         txSkelOuts =
-                          [ paysPK (wallet 1) (Script.assetClassValue ac1 1 <> Script.assetClassValue ac2 2),
-                            paysPK attacker (Script.assetClassValue ac1 1)
+                          [ wallet 1 `receives` Value (Script.assetClassValue ac1 1 <> Script.assetClassValue ac2 2),
```

Comment:
Is this equivalent to: 

```haskell
wallet 1 `receives` (Value (Script.assetClassValue ac1 1) <&&> Value (Script.assetClassValue ac2 2))
```

Reply:
Not anymore. The composition no longer allows to combine the content of values. It was controversial and for good reasons. Now you can only assign each component once at most (thanks to the type families constraining the types).